irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -97,91 +162,184 @@ namespace iroha {
           // Transaction counter
           + transaction_.quote(default_tx_counter) + ", "
           + transaction_.quote(account.json_data) + ");");
+
+      auto message_gen = [&] {
+        return (boost::format("failed to insert account, "
+                              "account id: '%s', "
+                              "domain id: '%s', "
+                              "quorum: '%d', "
+                              "transaction counter: '%d', "
+                              "json_data: %s")
```

Comment:
same for wrapping that field

Reply:
JSON data will be in the form "{...}", so any additional wrapping might be confusing